scripts/spellcheck.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+results=./Spellchecker.results
+> $results
+
+SpellcheckFile() {
+    filename=$1
+    extension=${filename##*.}
+    if [ $extension == "md" ]; then
```

Comment:
I think this could be easier by creating specific Makefile rules.

Reply:
I considered it. The extension is an argument that is necessary for the script, but why make it an argument when you can get this from the filename of the file you're currently checking. 

This way will make it simpler if I ever need to add new extensions to check, and it's not really that complicated to do one way or the other so I didn't stop at it that much.